std_msgs/Header header

interface/TensegrityBars[] trajectory
